docs/janssen-server/reference/json/properties/config-api-properties.md

```
@@ -1,6 +1,34 @@
+- properties
+---
+
+# Config-API Configuration Properties
+
+| Property Name | Description |  | 
+|-----|-----|-----|
+| disableExternalLoggerConfiguration | Choose whether to disable external log4j configuration override | [Details](#disableexternalloggerconfiguration) |
+| protectionMode | Protection mode for the Lock server (OAuth or Cedarling) | [Details](#protectionmode) |
+
+
+### disableExternalLoggerConfiguration
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)